onnxruntime/core/framework/allocator.cc

```diff
@@ -72,9 +72,9 @@ void AllocatorDefaultFree(void* p) {
 }
 
 #else
-void* AllocatorDefaultAlloc(size_t size) {
-  const size_t alignment = MlasGetPreferredBufferAlignment();
-  if (size <= 0) return nullptr;
+
+void* AllocatorDefaultAllocAligned(size_t size, size_t alignment) {
```

Comment:
Does there need to be a AllocatorDefaultAllocAligned in the `ifdef USE_MIMALLOC` section?